server/odc-service/src/main/java/com/oceanbase/odc/service/pldebug/PLDebugService.java

```diff
@@ -161,13 +170,21 @@ public String start(@NonNull ConnectionSession connectionSession, StartPLDebugRe
                     PLDebugConstants.PL_DEBUG_PACKAGE, ConnectionSessionUtil.getCurrentSchema(connectionSession), e);
             throw OBException.executeFailed(ErrorCodes.DebugPackageCreateFailed, e.getMessage());
         }
-        PLDebugSession plDebugSession = new PLDebugSession(authenticationFacade.currentUserId());
+        PLDebugSession plDebugSession =
+                new PLDebugSession(authenticationFacade.currentUserId(), this::generateSessionId);
         plDebugSession.setDbmsoutputMaxRows(sessionProperties.getDbmsOutputMaxRows());
         plDebugSession.start(connectionSession, debugSessionExecutor, req, sessionTimeoutSeconds, syncEnabled);
         debugId2Session.put(plDebugSession.getSessionId(), plDebugSession);
         return plDebugSession.getSessionId();
     }
 
+    private String generateSessionId() {
+        PLDebugSessionId id = new PLDebugSessionId();
+        id.setUuid(RandomStringUtils.random(10, UUID.randomUUID().toString().replace("-", "")));
```

Comment:
why not use uuid straightly? 
if set lengh as 10, may result into duplicated uuid.
may refer odc StringUtils.uuidNoHyphen straightly.

Reply:
done!